app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -49,23 +48,22 @@ static Dimelo setupDimelo(Context context) {
             dimelo.setUserIdentifier(userIdVal);
         }
 
-        if (isHmsAvailable(context)) {
-            dimelo.setPushNotificationService("hms");
-        } else {
-            FirebaseApp.initializeApp(context);
-            FirebaseMessaging.getInstance().getToken()
-                    .addOnCompleteListener(new OnCompleteListener<String>() {
-                        @Override
-                        public void onComplete(Task<String> task) {
-                            if (task.isSuccessful()) {
-                                String refreshedToken =  task.getResult();
-                                if (refreshedToken != null) {
-                                    Dimelo.getInstance().setDeviceToken(refreshedToken);
-                                }
-                            }
-                        }
-                    });
-        }
+        // if (isHmsAvailable(context)) {
+        //     dimelo.setPushNotificationService("hms");
+        // } else {
+        //     FirebaseApp.initializeApp(context);
+        //     FirebaseMessaging.getInstance().getToken().addOnCompleteListener(new OnCompleteListener<String>() {
+        //         @Override
+        //         public void onComplete(Task<String> task) {
+        //             if (task.isSuccessful()) {
+        //                 String refreshedToken =  task.getResult();
+        //                 if (refreshedToken != null) {
+        //                     Dimelo.getInstance().setDeviceToken(refreshedToken);
+        //                 }
+        //             }
+        //         }
+        //     });
+        // }
```

Comment:
the main changes are here, I changed the indentation a bit so that it doesn't end up 90 columns to the right and I commented everything so that we can compile the sample app without having to add all of these FCM/HMS related stuff

the rest of the changes are indentation fix